### PR TITLE
Updated to use the new design: most cited works in venue

### PR DESCRIPTION
**Pull request by egonw**
Is there an easy way for me to test this locally?

### CODE REVIEW

**Comment by fnielsen** [comment]
What do you mean by "test this locally"? You can do python runserver.py and the point your web browser to the displayed URL.

**Comment by egonw** [comment]
Yes, like that... silly me, I overlooked that script... I got it set up now, so I can start making proper patches. Please hold of this one until I tested it.

**Comment by egonw** [comment]
OK, confirmed working locally. The patch is good to go.

**Comment by egonw** [comment]
Oh, let me do the next table too...

**Comment by egonw** [comment]
OK, one table left to upgrade...

File: scholia/app/templates/venue.html [hunk 1]
```diff
@@ -39,7 +39,7 @@
 
  prolificAuthorsSparql = `
 # Prolific authors for a specific journal
-SELECT ?count ?author ?authorLabel ?orcid
+SELECT ?count ?author ?authorLabel ?ORCID
```
Comment (fnielsen) [inline scholia/app/templates/venue.html]: I usually use lower case for variables, - even variables that refer to identifiers that are usually written with capital letter. In this case it does not work with scholia.js: https://github.com/fnielsen/scholia/blob/master/scholia/app/static/scholia.js#L27

Comment (fnielsen) [inline scholia/app/templates/venue.html]: Ok I see that the header in the table on does not uppercase here: https://tools.wmflabs.org/scholia/venue/Q6294930#Prolific%20authors

File: scholia/app/templates/venue.html [hunk 2]
```diff
@@ -84,14 +84,24 @@
...
  `
 
-
- 
+ mostCitedWorksSparql = `
+  # Most cited works published in venue
+  SELECT (COUNT(?citing_work) AS ?count) ?work ?workLabel  WHERE {
+    ?work wdt:P1433 wd:Q6294930.
+    ?citing_work wdt:P2860 ?work.
+    SERVICE wikibase:label { bd:serviceParam wikibase:language "en,da,de,es,fr,jp,nl,no,ru,sv,zh" . }
```
Comment (egonw) [inline scholia/app/templates/venue.html]: Yes, I saw that... they're also more complicated, but this is the kind of review I was hoping for. I will update that patch accordingly!

File: scholia/app/templates/venue.html (hunk 1 above)
Comment (egonw) [inline scholia/app/templates/venue.html]: Should I change back the variable name?

Do I understand correctly, we can upper case the table column headers later?

Comment (fnielsen) [inline scholia/app/templates/venue.html]: Yes, change it back for now. We should fix the header at some point. There is also the problem of generated a link when the cell is empty

File: scholia/app/templates/venue.html (hunk 2 above)
Comment (egonw) [inline scholia/app/templates/venue.html]: This one is still pending...

**Comment by egonw** [comment]
@fnielsen I've updated the query to use the nested select structure for the labels now.

File: scholia/app/templates/venue.html [hunk 3]
```diff
@@ -84,14 +84,75 @@
  LIMIT 200
  `
 
+ citingVenuesSparql = `
+#defaultView:Table
+# Venues citing articles from this specific journal
+SELECT ?count ?citing_journal ?citing_journalLabel 
+WITH {
+  SELECT (COUNT(?citing_work) as ?count) ?citing_journal WHERE {
+    ?work wdt:P1433 wd:Q6294930 .
```
Comment (fnielsen) [inline scholia/app/templates/venue.html]: This needs to be parametrized

File: scholia/app/templates/venue.html [hunk 4]
```diff
@@ -84,14 +84,75 @@
...
+ORDER BY DESC(?count)
+  `
+
+ mostCitedWorksSparql = `
+#defaultView:Table
+# Most cited works published in venue
+SELECT ?count ?work ?workLabel
+WITH {
+  SELECT (COUNT(?citing_work) AS ?count) ?work WHERE {
+    ?work wdt:P1433 wd:Q6294930.
```
Comment (fnielsen) [inline scholia/app/templates/venue.html]: Needs parameterization

File: scholia/app/templates/venue.html [hunk 5]
```diff
@@ -84,14 +84,75 @@
...
+  `
 
- 
+ mostCitedAuthorsSparql = `
+  # Most cited authors published in venue
+SELECT ?count ?author ?authorLabel ?orcid
+WITH {
+  # Count the number of works author by the author publishing in the journal
+  SELECT ?author (COUNT(?work) AS ?count) WHERE {
+    ?work wdt:P1433 wd:Q6294930 .
```
Comment (fnielsen) [inline scholia/app/templates/venue.html]: Parameterization

File: scholia/app/templates/venue.html (hunk 3 above)
Comment (egonw) [inline scholia/app/templates/venue.html]: I have no idea what that means... can you give me a pointer?

**Comment by fnielsen** [comment]
With 5d720bcec8f31342cd56568b83e98ab82e40940f I fixed the parametrization

**Comment by egonw** [comment]
Carp... sorry, very embarrassing...

**Comment by fnielsen** [comment]
Now in production

**Comment by fnielsen** [comment]
No problem :)

**Comment by fnielsen** [comment]
Sorry for the delay. I was traveling.

**Pull request merged**